cdap-ui/app/cdap/components/NamespaceDetails/NamespaceDetails.scss

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../styles/variables.scss";
+
+.namespace-details-container {
+  /* 50 px top panel, 54px footer */
+  height: calc(100vh - 50px - 104px);
```

Comment:
Is this necessary? we already have `#app-container` with height `calc(100vh - 104px)` right? Until `Home` I see all components taking the height of the parent.